python/ray/_private/worker.py

```diff
@@ -2194,6 +2194,12 @@ def disconnect(exiting_interpreter=False):
         ray_actor._ActorClassMethodMetadata.reset_cache()
 
 
+def start_import_thread(worker=global_worker):
+    worker.function_actor_manager = FunctionActorManager(worker)
```

Comment:
Can you help me understand why `function_actor_manager` needs to change?

Reply:
ah good catch